ext/standard/tests/general_functions/gh9589.phpt

```diff
@@ -0,0 +1,10 @@
+--TEST--
+dl() segfaults when module is already loaded
+--EXTENSIONS--
+dl_test
```

Comment:
It's probably sensible to add a comment regarding the usage of `dl_test`; actually, it doesn't matter which extension we use for the test, but it is mandatory that it is a shared module; this is always given for `dl_test`.